### PR TITLE
Added confirmation of success or error.

### PR DESCRIPTION
**Pull request by FelixLetheren**

### CODE REVIEW

File: newFish.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
     && isset($_POST['color'])
     && isset($_POST['pattern'])
 ) {
-    insertEntryIntoDB($db,$_POST);
+   echo inputConfirmation(insertEntryIntoDB($db,$_POST));
```
Comment (mallowmew) [inline newFish.php]: Consider loading this string into a variable and then echoing the variable in your pages' HTML body. The top-of-document echo is quite difficult to do anything with styling-wise.

File: test/FunctionsTest.php [hunk 2]
```diff
@@ -69,4 +68,23 @@ public function testDisplayFish_throwsTypeErrorWithWrongInputType(){
...
     }
+
+    public function testInputConfirmation_returnsSuccessMessageOnTrue(){
+
+        $expectedresult = '<h2 class="success">Success! Fish has been inserted into collection</h2>';
+
+        $result = inputConfirmation(true);
+
+        $this->assertEquals($expectedresult, $result);
+
```
Comment (sjr-mayden) [inline test/FunctionsTest.php]: Extra newline

File: test/FunctionsTest.php [hunk 3]
```diff
@@ -69,4 +68,23 @@ public function testDisplayFish_throwsTypeErrorWithWrongInputType(){
...
+    }
+    public function testInputConfirmation_returnsFailureMessageOnFalse(){
+
+        $expectedresult = '<h2 class="failure">Oops! You\'ve made and error. Please check you\`ve correctly filled all fields!</h2>';
+
+        $result = inputConfirmation(false);
+
+        $this->assertEquals($expectedresult, $result);
+
```
Comment (sjr-mayden) [inline test/FunctionsTest.php:97]: Extra newline

File: test/FunctionsTest.php [hunk 4]
```diff
@@ -69,4 +68,23 @@ public function testDisplayFish_throwsTypeErrorWithWrongInputType(){
...
+    public function testInputConfirmation_returnsSuccessMessageOnTrue(){
+
+        $expectedresult = '<h2 class="success">Success! Fish has been inserted into collection</h2>';
+
+        $result = inputConfirmation(true);
+
+        $this->assertEquals($expectedresult, $result);
+
+    }
+    public function testInputConfirmation_returnsFailureMessageOnFalse(){
```
Comment (sjr-mayden) [inline test/FunctionsTest.php]: `{` should go on a new line

File: test/FunctionsTest.php [hunk 5]
```diff
@@ -69,4 +68,23 @@ public function testDisplayFish_throwsTypeErrorWithWrongInputType(){
 
         displayFish($examplefish);
     }
+
+    public function testInputConfirmation_returnsSuccessMessageOnTrue(){
```
Comment (sjr-mayden) [inline test/FunctionsTest.php]: `{` should go on a new line

Comment (sjr-mayden) [reply test/FunctionsTest.php]: + one more in `testDisplayFish_throwsTypeErrorWithWrongInputType`

File: index.php [hunk 6]
```diff
@@ -26,5 +26,6 @@
         <img src="images/plus-icon.png" alt="Add a new fish">
     </a>
 </div>
+
```
Comment (sjr-mayden) [inline index.php]: Unnecessary newline

File: newFish.php [hunk 7]
```diff
@@ -50,5 +50,8 @@
         <img src="images/return.png" alt="Return to Collection">
     </a>
 </div>
+<div class="content-container">
+    <?php echo $message;?>
```
Comment (sjr-mayden) [inline newFish.php]: This `echo`s regardless of whether anything has been set

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

**Comment by sjr-mayden** [comment]
If I can't add a fish picture then there should be a placeholder of some sort so that you don't break your front end design

**Pull request merged**